src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,142 @@
+                variant="contained"
+                color="primary"
+                startIcon={<FontAwesomeIcon icon={faPlus} />}
+              >
+                Add Action Node
+              </Button>
+            </td>
+            <td className={styles.description}>
+              Any clinical or worfklow step which is not a decision.
+            </td>
+          </tr>
+          <tr>
+            <td className={styles.button}>
+              <Button
+                variant="contained"
+                color="primary"
+                startIcon={<FontAwesomeIcon icon={faPlus} />}
+              >
+                Add Decision Node
+              </Button>
+            </td>
+            <td className={styles.description}>
+              A logical branching point based on clinical or workflow criteria.
+            </td>
+          </tr>
+          <tr>
+            <td className={styles.button}>
+              <Button
+                variant="contained"
+                color="primary"
+                startIcon={<FontAwesomeIcon icon={faPlus} />}
+              >
+                Add Reusable Node
+              </Button>
+            </td>
+            <td className={styles.description}>
+              A previously built node or group of nodes defining a set of criteria.
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  );
+};
+
+const SidebarHeader: FC<SidebarHeaderProps> = ({ currentNode }) => {
+  return (
+    <div className={styles.header}>
+      <div className={styles.icon} id={styles.back}>
+        <FontAwesomeIcon icon={faChevronLeft} />
+      </div>
+      <div className={styles.nodeName}>{currentNode}</div>
+      <div className={styles.icon}>
+        <FontAwesomeIcon icon={faEdit} />
+      </div>
+      <div className={styles.icon} id={styles.nodeSettings}>
+        <FontAwesomeIcon icon={faEllipsisH} />
+      </div>
+    </div>
+  );
+};
+
+const Sidebar: FC<SidebarProps> = ({ headerElement }) => {
+  const sidebarContainerElement = useRef<HTMLDivElement>(null);
+  const [isExpanded, setIsExpanded] = useState<boolean>(true);
+  const [currentNode] = useState<string>('Start');
+
+  const expand = (): void => {
+    setIsExpanded(!isExpanded);
+  };
+
+  // Set the height of the sidebar container
+  useEffect(() => {
+    if (sidebarContainerElement?.current && headerElement?.current)
+      sidebarContainerElement.current.style.height =
+        window.innerHeight - headerElement.current.clientHeight + 'px';
+  }, [isExpanded, headerElement]);
+
+  if (isExpanded) {
+    return (
+      <div className={styles.sidebarContainer} ref={sidebarContainerElement}>
+        <div className={styles.sidebar}>
+          <SidebarHeader currentNode={currentNode} />
+          <hr />
+          <DropDown
+            label={'Node Type'}
+            id={'Node Type'}
+            options={[
+              { label: 'Action', value: 'action' },
+              { label: 'Branch', value: 'branch' }
```

Comment:
These should probably be consistent with our buttons. My design has "step" and "decision" for node types, terms I grabbed from another pathway builder. I'm ok with whatever terms we decide on, but they should be consistent. Currently our buttons show "action" and "decision".

Reply:
I'd prefer to avoid the term "decision" because in a vacuum the software developers and clinicians expect opposite things from that term. Jim has referred to our action nodes as "decision nodes" because the clinician has to make a decision as to whether to accept or decline. "Branch" is less ambiguous.

Reply:
That's my bad I meant to bring that up to you. I went with 'Action' and 'Branch' because I was personally confused about 'step' and 'decision' and in design conversations we have always referred to them as action (or sometimes guidance) and branch nodes

Reply:
I'm fine with "Action" and "Branch" so let's fix the buttons to match. Thanks!

Reply:
Updated the Decision button to say Branch